### ISSUE TITLE
Feature Request: Be able to export the whole mesh (model) instead of just single meshes

### ISSUE THREAD

**Issue by Kubsy** [MontyTRC89/Tomb-Editor#539]
I haven't found an option to export whole models from wadtool, only just single meshes. It would be nice to be able to export a whole model.

**Comment by Castigavi** [comment]
I agree that this is something that is really needed, since the only other tool that can do that is TRViewer and it's honestly utter shite.
